tests/config/basic/butler.yaml

```diff
@@ -4,7 +4,7 @@ datastore:
   root: ./butler_test_repository
   templates:
     default: "{datasetType}/{tract:?}/{patch:?}/{filter:?}/{camera:?}_{visit:?}"
-    calexp: "{datasetType}.{component:?}/{datasetType}_v{visit}_f{filter}_{component:?}"
+    calexp: "{datasetType}.{component:?}/{datasetType}_v{visit}_f{filter:?}_{component:?}"
```

Comment:
This may be fine for a test, but real-world calexps should have `sensor` and should not *need* to have `filter`.

Reply:
The problem with `calexp` was not for the test, but rather for `ci_hsc` where `sensor` does not get passed I think.

Reply:
I can remove it here and (temporarily) override it in the `ShimButler` config.

Reply:
I don't this config is used at all in the ci_hsc conversion.  The one at `config/datastores/posixDatastore.yaml` is what you need to modify to have any effect on ci_hsc.

Reply:
Sorry, I wasn't specific enough, it isn't used in the conversion, but it is used by the `ShimButler`. Anyway, I'll revert the change here.